src/dev/platform/f3xx/IWDGf3xx.cpp

```diff
@@ -12,12 +12,17 @@ IWDGf3xx::IWDGf3xx(uint32_t ms) : halIWDG{
         ms / 8 - 1,
         ms / 8 - 1,
     },
-} {
+} {}
+
+void IWDGf3xx::init() {
     HAL_IWDG_Init(&halIWDG);
+    isActive = true;
```

Comment:
Should any error handling be done here? I.E check HAL return code and return our own EVT-Core status code

Reply:
There probably should be some error handling, but that is going to have to be addressed across EVT-core, so I'd rather not hold this PR up for it